### PR TITLE
Enhance API of `ServiceMessage` and `ClientMessage`

### PR DESCRIPTION
**Pull request by dmytro-lazebnyi**

### CODE REVIEW

File: services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java [hunk 1]
```diff
@@ -162,11 +162,33 @@ public boolean hasData(Class<?> dataClass) {
     }
   }
 
+  /**
+   * Describes whether the client message is an error.
```
Comment (artem-v) [inline services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java]: Just _message_ why exactly _client_ ?

File: services-gateway-clientsdk/src/main/java/io/scalecube/services/gateway/clientsdk/ClientMessage.java [hunk 2]
```diff
@@ -72,11 +72,33 @@ public boolean hasData(Class<?> dataClass) {
     return dataClass.isPrimitive() ? hasData() : dataClass.isInstance(data);
   }
 
+  /**
+   * Describes whether the client message is an error.
```
Comment (artem-v) [inline services-gateway-clientsdk/src/main/java/io/scalecube/services/gateway/clientsdk/ClientMessage.java]: Just message, why exactly _clietn_ ?

**Review by artem-v — CHANGES_REQUESTED** [review]

File: services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java (hunk 1 above)
Comment (dmytro-lazebnyi) [reply services-api/src/main/java/io/scalecube/services/api/ServiceMessage.java]: Fixed.

File: services-gateway-clientsdk/src/main/java/io/scalecube/services/gateway/clientsdk/ClientMessage.java (hunk 2 above)
Comment (dmytro-lazebnyi) [reply services-gateway-clientsdk/src/main/java/io/scalecube/services/gateway/clientsdk/ClientMessage.java]: Fixed.

**Pull request merged**